Apps/phconsolidatedscreeninglist/consolidatedscreeninglist.json

```diff
@@ -4,47 +4,45 @@
     "description": "US Companies that develop products including encryption are required to validate users against various screens before distributing the product. This helps prevent, for example, sending a weapons shipment to North Korea. Or, perhaps some kick-ass security automation software.",
     "type": "information",
     "product_vendor": "Splunk",
-    "logo": "consolidatedscreeninglist.png",
-    "logo_dark": "consolidatedscreeninglist_dark.png",
+    "logo": "logo_splunk.svg",
```

Comment:
We should use the consolidated screening list logo. If one is not available, you can ask for one to be created in #phantom-ui channel in Slack